src/inputs/dropdown/dropdown.jsx

```diff
@@ -406,24 +437,36 @@ class Dropdown extends React.Component {
     }
 
     _menuRenderer(params) {
-        const items = _.map(params.options, (o, i) => {
-            if (this.props.selectionOptionComponent) {
-                const OptionComponent = this.props.selectionOptionComponent;
+        const { selectionOptionComponent } = this.props;
+        const { focusedOption, onFocus, options, selectValue, valueArray, valueKey } = params;
```

Comment:
I believe these things destructured from `params` should be on separate lines.

Reply:
I agree, NAB though.

Reply:
Looks good.  Resolving.